aziotctl/aziotctl-common/src/config/super_config.rs

```diff
@@ -317,6 +322,26 @@ pub enum CertIssuanceMethod {
     SelfSigned,
 }
 
+fn deserialize_check_protocol<'de, D>(de: D) -> Result<Option<url::Url>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let opt = Option::<Url>::deserialize(de)?;
+    match &opt {
+        Some(url) => {
+            if url.scheme() == "http" {
+                eprintln!(
+                    "Warning: EST server URL {:?} is configured with unencrypted HTTP, which may expose device to man-in-the-middle attacks.
+                    To clear this warning, configure HTTPS for your EST server and update the URL.",
```

Comment:
When printed, this string will have the line break and prefixing spaces left in. So, it would appear as
```
Warning: EST server URL {:?} is configured with unencrypted HTTP, which may expose device to man-in-the-middle attacks.
                    To clear this warning, configure HTTPS for your EST server and update the URL.
```
Is this intentional?